src/components/LoadingMessage/LoadingMessage.jsx

```diff
@@ -0,0 +1,87 @@
+import _ from 'lodash';
+import React from 'react';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createClass, findTypes } from '../../util/component-types';
+import LoadingIcon from '../Icon/LoadingIcon/LoadingIcon';
+
+const cx = lucidClassNames.bind('&-LoadingMessage');
+
+const {
+	node,
+	string,
+} = React.PropTypes;
+
+/**
+ *
+ * {"categories": ["communication"], "madeFrom": ["LoadingIcon"]}
+ *
+ * A loading message.
+ *
+ */
+const LoadingMessage = createClass({
+	propName: 'LoadingMessage',
```

Comment:
i think it's safe to remove the propName here, the only valid value you can pass in as a prop is if it's `{<Element>}`, and at that point, might as well just pass it thru children.